bin/migration-scripts/lts-to-lts/0.16-to-0.23/playbooks/tasks/rollback.yaml

```diff
@@ -234,6 +234,16 @@
     db: "{{ database_name }}_houston"
     query: "drop table migrations;"
 
+# Delete Scheduler deployment before rollback
+# to prevent init container wait-for-airflow-migrations 
+# to append in existing 0.15.7 airflow chart preventing 
+# creation of 2 init containers (run-airflow-migrations + wait-for-airflow-migrations)
```

Comment:
@shmanu017 I feel this comment is not clearly describing what the intention of this section is. Can you take another stab at it please?

Reply:
@vishwas-astro Please review the new description

Reply:
I think the old wording was better. How about:

```
# Delete scheduler deployment before rollback, in order to prevent the init container 'wait-for-airflow-migrations' from __________
```